examples/dynamic-config-cp/Dockerfile-control-plane

```diff
@@ -1,7 +1,7 @@
 FROM golang
 
 RUN apt-get -y update && apt-get install -y -qq --no-install-recommends netcat
-RUN git clone https://github.com/envoyproxy/go-control-plane
+RUN git clone -b udpa-xds https://github.com/phlax/go-control-plane
```

Comment:
this wont be necessary if https://github.com/envoyproxy/go-control-plane/pull/404 lands

Reply:
Merged